Packs/Core/Playbooks/playbook-WildFire_Malware_README.md

```diff
@@ -0,0 +1,85 @@
+This playbook handles WildFire Malware alerts.
+It performs enrichment on the different alert entities and establishes a verdict.
+For a possible true positive alert, the playbook performs further investigation for related IOCs and executes a containment plan.
+
+## Malware Investigation and Response Flow
+
+An attacker may use malware to gain control, gather sensitive information, and disrupt a system's normal operations.
+
+**Investigative Actions:**
+
+Cortex XDR investigates the file hashes, IP addresses, and identities involved in the detected activity using the **Enrichment for Verdict** and **Endpoint Investigation Plan** playbooks.
+
+**Response Actions**   
+Phase 1:   
+The WildFire Malware playbook's first response actions are based on the data available within the alert.  
+In this phase, the playbook  executes the **Containment Plan** playbook with the file hash as an input.  
+Phase 2:  
+When the playbook continues, if other suspicious IOCs are found in the investigation, the next phase of response is executed, which includes the **Containment Plan** playbook with the found IOCs as inputs.
+
```

Comment:
For each phase - let's explain which of the containment actions are by default true to run automatically